kafka-ui-react-app/src/redux/reducers/schemas/selectors.ts

```diff
@@ -24,6 +34,22 @@ export const getSchemaList = createSelector(
     if (!isFetched) {
       return [];
     }
-    return allNames.map((subject) => byName[subject]);
+    return allNames.map((subject) => byName[subject as string]);
   }
 );
+
+const getSchemaName = (_: RootState, subject: string) => subject;
+
+export const getSchema = createSelector(
+  getSchemaMap,
+  getSchemaName,
+  (schemas, subject) => schemas[subject]
+);
+
+export const getSchemaVersions = createSelector(
```

Comment:
If we want this as the default behaviour, we probably should put in a sorted format in the store. Otherwise, better to rename it to `getSortedSchemaVersions` to be more clear.